docs/reference/units.ipynb

```diff
@@ -448,6 +471,192 @@
+    "  \n",
+    "## Prioritizing Units in String Formatting\n",
+    "\n",
+    "When dealing with crystals or molecules, it is often convenient to use angstrom as a unit.\n",
+    "But by default, string formatting tends to prefer different bases in composite units.\n",
+    "For example"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.Unit('us/angstrom**2')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This result is not very useful.\n",
+    "We can make the formatter prefer angstrom by defining it as an alias of itself (or alternatively of `'10^-10 m'`):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.units.aliases['angstrom'] = 'angstrom'\n",
+    "sc.Unit('us/angstrom**2')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Aliases are global and stay in effect until they are removed.\n",
+    "This can be done using `sc.units.aliases.clear()` to remove all aliases or, to remove only one, using"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "del sc.units.aliases['angstrom']\n",
+    "sc.Unit('us/angstrom**2')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Alternatively, a context manager can be used to remove aliases automatically:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with sc.units.aliases.scoped(angstrom='angstrom'):\n",
+    "    print(sc.Unit('us/angstrom**2'))\n",
+    "print(sc.Unit('us/angstrom**2'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "But note that the context manager uses the global alias table and affects code outside of the context.\n",
+    "Details are explained in [scipp.units.aliases.scoped](../generated/modules/scipp.units.UnitAliases.rst#scipp.units.UnitAliases.scoped).\n",
+    "\n",
+    "See [scipp.units.UnitAliases](../generated/modules/scipp.units.UnitAliases.rst) for the full API of `sc.units.aliases`.\n",
+    "\n",
+    "## Defining New Units\n",
+    "\n",
+    "It is possible to define completely new units as aliases as long as they can be expressed in terms of other units.\n",
+    "For example, define the horse length:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.units.aliases['horse'] = sc.scalar(2.4, unit='m')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Alternatively, we can pass `'2.4 m'` instead of a variable.\n",
+    "\n",
+    "The string formatter now prioritizes 'horse' over 'm' when appropriate:"
```

Comment:
It might be confusing for users that are not aware of units scales... can you
1. give an example (or all examples?) of formatting *variables* with this unit, so they see the value in the variable as well as the scale factor.
2. Show as counterexample that `sc.scalar(2.4, 'm')` does *not* format as `'horse'`?